_core/smart/tutorial.md

```diff
@@ -1,211 +1,188 @@
+> To create private functions, you would use the `define-private` keyword. Private functions can only be executed by the current smart contract - not from other smart contracts published to the same network. Only public functions can be called from other contracts.
 
-    The function doesn't take any parameters and simply returns "hello world" using the [`ok`](https://docs.blockstack.org/core/smart/clarityref#ok) response constructor.
+The function doesn't take any parameters and simply returns "hello world" using the [`ok`](https://docs.blockstack.org/core/smart/clarityref#ok) response constructor.
 
-    Let's review the second public function, `echo-number`. As opposed to the function before, this takes an input parameter of the type [`int`](https://docs.blockstack.org/core/smart/clarityref#int-type). Along with integer, Clarity supports the following types:
-   * [uint](https://docs.blockstack.org/core/smart/clarityref#uint-type): 16-byte unsigned integer
-   * [principal](https://docs.blockstack.org/core/smart/clarityref#principal-type): spending entity, roughly equivalent to a Stacks address
-   * [boolean](https://docs.blockstack.org/core/smart/clarityref#bool-type): `true` or `false`
-   * [buffer](https://docs.blockstack.org/core/smart/clarityref#buffer-type): fixed-length byte buffers
-   * [tuple](https://docs.blockstack.org/core/smart/clarityref#tuple-type): named fields in keys and values
+Let's review the second public function, `echo-number`. As opposed to the function before, this takes an input parameter of the type [`int`](https://docs.blockstack.org/core/smart/clarityref#int-type). Along with integer, Clarity supports the following types:
+* [uint](https://docs.blockstack.org/core/smart/clarityref#uint-type): 16-byte unsigned integer
+* [principal](https://docs.blockstack.org/core/smart/clarityref#principal-type): spending entity, roughly equivalent to a Stacks address
+* [boolean](https://docs.blockstack.org/core/smart/clarityref#bool-type): `true` or `false`
+* [buffer](https://docs.blockstack.org/core/smart/clarityref#buffer-type): fixed-length byte buffers
+* [tuple](https://docs.blockstack.org/core/smart/clarityref#tuple-type): named fields in keys and values
 
-    The function simply uses the `ok` response and returns the value passed to the function.
+The function simply uses the `ok` response and returns the value passed to the function.
 
-## Step 3: Running tests
+## Access the Explorer Sandbox
 
-The starter project comes with test tooling already set up for you (using [Mocha](https://mochajs.org/)). Let's run the tests and review the results:
+{% include note.html content="This tutorial uses a developer preview release of the <a href='https://testnet-explorer.blockstack.org/'>Stacks 2.0 Explorer</a>. Please feel free to report issues or request enhancements on the <a href='https://github.com/blockstack/explorer/issues/new'>blockstack/explorer</a> repository." %}
```

Comment:
I think we're already using the phrase "preview release" and instructing people to "report issues" so that expectation is well set. But definitely like the idea of adding the link to the forum post; done.